helix-core/src/main/java/org/apache/helix/controller/stages/IntermediateStateCalcStage.java

```diff
@@ -31,6 +31,7 @@
 
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
+
```

Comment:
@desaikomal It occurred to me that you might not be using helix-style-intellij.xml to format the code. It's a requirement for Helix PRs, and you can find guidelines here: https://github.com/apache/helix/wiki/Helix-Code-Style-in-Intellij-IDE :)